Website/Providers/DataProviders/SqlDataProvider/07.04.00.SqlDataProvider

```diff
@@ -691,7 +695,7 @@ BEGIN
         'Reject',
         'Reject',
         NULL,
-        1,
+        2,
```

Comment:
Why are you changing the order from 1 to 2 for this and the discard action later?